ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs

```diff
@@ -362,21 +370,23 @@ instance ConfigSupportsNode (ShelleyBlock era) where
   getNetworkMagic = shelleyNetworkMagic
 
 {-------------------------------------------------------------------------------
-  RunNode instance
+  InitNodeStorage instance
 -------------------------------------------------------------------------------}
 
-instance TPraosCrypto era => RunNode (ShelleyBlock era) where
+instance TPraosCrypto era => InitNodeStorage (ShelleyBlock era) where
   -- We fix the chunk size to 10k
   nodeImmutableDbChunkInfo =
-      simpleChunkInfo
-    . EpochSize
-    . (* 10)
-    . maxRollbacks
-    . tpraosSecurityParam
-    . tpraosParams
-    . configConsensus
-
-  nodeCheckIntegrity cfg = verifyBlockIntegrity tpraosSlotsPerKESPeriod
-    where
-      TPraosParams { tpraosSlotsPerKESPeriod } =
-        tpraosParams $ configConsensus cfg
+        simpleChunkInfo
+      . EpochSize
+      . (* 10)
```

Comment:
Document why: to align with what mainnet is doing (this will only be used for Shelley-only testnets).